src/components/VHeader/VHeader.vue

```diff
@@ -149,12 +147,12 @@ const VHeader = defineComponent({
      * Shows the loading state or result count.
      */
     const searchStatus = computed(() => {
-      if (!isSearchRoute.value || store.state.search.query.q === '') return ''
+      if (!isSearchRoute.value || searchStore.query.q === '') return ''
       if (isFetching.value) return i18n.t('header.loading')
       return getI18nCount(resultsCount.value)
     })
 
-    const localSearchTerm = ref(store.state.search.query.q)
+    const localSearchTerm = ref(searchStore.query.q)
     const searchTerm = computed({
```

Comment:
We don't appear to need the distinction between `localSearchTerm` and `searchTerm` any more, do we? `searchTerm` just gets and sets `localSearchTerm`'s value anyway.

Reply:
Yes, I updated the handling of the `searchTerm`